src/fineImg.js

```diff
@@ -0,0 +1,24 @@
+const giphy = document.getElementById('giphy');
+
+const changeBackground = (gif) => {
+  giphy.style.background = `url(${gif})`;
+  giphy.style.backgroundSize = 'cover';
+  giphy.style.backgroundRepeat = 'no-repeat';
+};
+
+const fineImg = (weather) => {
+  const cli = weather.replace(' ', '-');
+  fetch(`https://api.unsplash.com/search/photos?query=weather-${cli}&client_id=qO5N_GGS-NvGLQ9-5mQhX99at1FNPmBDCSZDmK6TOdk`, { mode: 'cors' })
+    .then((response) => response.json())
+    .then((gif) => {
+      console.log(gif);
+      const select = gif.results[Math.floor(Math.random() * gif.results.length)];
+      changeBackground(select.urls.full);
+    })
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
It is not a good practice to have `console.log` statements committed to your repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png) from Pathwright)